src/app/containers/MediaPlayer/index.jsx

```diff
@@ -151,6 +153,11 @@ const MediaPlayerContainer = ({
     );
   }
 
+  logEmbedSourceStatus({
+    url: embedSource,
```

Comment:
Could we rename `url` -> `embedUrl`?
Then, can we pass in the asset ID as the `url` -> I think this should be the `assetId` prop

This is just for consistency with the rest of the logging across the application and means the logs will look something like this:
```
{event: media_player_info,
  message: {
    url: /pidgin/23248703,
    embedUrl: https://polling.test.bbc.co.uk/ws/av-embeds/cps/pidgin/23248703/p01kx42v/pcm,
    status: 200,
  }
}
```

Reply:
I feel this makes it a little ambiguous which URL the status is for. Actually, on first guess, it seems the status is for the url and not the embedUrl, which is misleading.

Reply:
I guess this would just be inconsistent with the other log messages throughout the application - we use url for the assetId elsewhere.

Reply:
Got you. Makes sense.